intel-sgx/scripts/gdb.py

```diff
@@ -44,6 +67,10 @@ def find_vma_base(addr):
         return start-offset
   return None
 
+def find_vma_base(addr):
+  # find_vma_base_from_proc_mappings(addr)  # old ftxsgx-runner, see python doc for details
+  return find_vma_base_from_runner(addr)  # new ftxsgx-runner, see python doc for details
```

Comment:
I'll change this after we have a solidly working ftxsgx-runner function, and then we'll be able to do "try function, otherwise fall back to proc mappings"

Reply:
Addressed now!